themes/bootstrap3/js/record.js

```diff
@@ -34,6 +34,40 @@ function checkRequestIsValid(element, requestType, icon = 'place-hold') {
     });
 }
 
+function wayfinderPlacementLinkLookup(element, icon = 'fa fa-map-marker') {
```

Comment:
We just recently adopted VuFind's icon helper. Please adjust to use it (`VuFind.icon()`, theme.config.php, finna-js-icons.phtml). See https://github.com/NatLibFi/NDL-VuFind2/commit/8304be79239d664a865eaa5dde77480c29d2a579 for examples.